de.po

```diff
@@ -2451,19 +2451,19 @@ msgstr "Der Virtualisierungsdienst (libvirt) ist nicht aktiv"
 #: src/components/vm/overview/bootOrder.jsx:95
 #: src/components/create-vm-dialog/createVmDialog.jsx:685
 msgid "Volume"
-msgstr "Lautstärke"
+msgstr "Datenträger"
 
 #: src/components/storagePools/storageVolumeCreate.jsx:83
 msgid "Volume failed to be created"
-msgstr ""
+msgstr "Fehler beim erstellen des Datenträgers"
 
 #: src/components/storagePools/createStoragePoolDialog.jsx:168
 msgid "Volume group name"
-msgstr ""
+msgstr "Datenträger Gruppenname"
```

Comment:
IMHO that's too confusing -- "volume group" is an LVM term. Cockpit's German translations use "Datenträgerverbund", but just keeping the term may be less confusing for admins. "Name der Volume-Group"?